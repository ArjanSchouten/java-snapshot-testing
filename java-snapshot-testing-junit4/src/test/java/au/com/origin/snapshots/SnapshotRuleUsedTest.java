package au.com.origin.snapshots;

import org.junit.ClassRule;
import org.junit.Test;

public class SnapshotRuleUsedTest {

    @ClassRule
    public static SnapshotRule snapshotRule = new SnapshotRule();

    @Test
    public void shouldUseExtension() {
        SnapshotMatcher.expect("Hello Wolrd").toMatchSnapshot();
    }

    @Test
    public void shouldUseExtensionAgain() {
        SnapshotMatcher.expect("Hello Wolrd Again").toMatchSnapshot();
    }
}
