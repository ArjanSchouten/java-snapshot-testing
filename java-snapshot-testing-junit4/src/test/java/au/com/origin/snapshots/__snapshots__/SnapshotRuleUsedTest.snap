au.com.origin.snapshots.SnapshotRuleUsedTest.shouldUseExtensionAgainViaInstanceVariable=[
Hello World
Hello World Again
]


au.com.origin.snapshots.SnapshotRuleUsedTest.shouldUseExtensionViaInstanceVariable=[
Hello World
]


hello_world=[
Hello World
]